pyproject.toml

```diff
@@ -1,3 +1,7 @@
+[build-system]
```

Comment:
This also makes the package pep517/518 compliant by declaring the build system and build dependencies. Modern pip "knows" to install this and call the right build backend.